test/uk/gov/hmcts/contino/MockDocker.groovy

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.contino
```

Comment:
contino? i guess because it maps to the Docker class

Reply:
yeah 😢